interfaces/apparmor/template.go

```diff
@@ -248,6 +249,7 @@ var defaultTemplate = []byte(`
   /etc/{,writable/}timezone r,
   @{PROC}/@{pid}/io r,
   owner @{PROC}/@{pid}/limits r,
+  @{PROC}/@{pid}/comm r,
```

Comment:
Can you change this to: `owner @{PROC}/@{pid}/comm r,` and move it to under this rule: `owner @{PROC}/@{pid}/cmdline r,` (it suffers from the same issues as /proc/*/cmdline and should be grouped there).